### DISCUSSION TITLE
Allow `using variable;` syntax

### DISCUSSION THREAD

**Discussion by maxkoshevoi** [dotnet/csharplang#6169] [Language Ideas] [answered]
Currently `using (variable) {}` is legal, but `using variable;` isn't.
It makes things like this harder to write:

```cs
IConnection connection;
if (_rabbitMqOptions.IsCluster)
{
    connection = factory.CreateConnection(_rabbitMqOptions.NodesAdresses);
}
else
{
    connection = factory.CreateConnection();
}

using connection; // CS0118 compilation error
using var channel = connection.CreateModel();

...
```

```cs
...

using (connection) // Legal syntax
{
    using var channel = connection.CreateModel();
    ...
}
```

**Comment by svick** [comment, marked as answer]
This is a duplicate of https://github.com/dotnet/csharplang/discussions/2593.

Though based on the discussion at https://github.com/dotnet/csharplang/issues/2235, it seems the `using connection;` syntax is not going to be allowed, but the more verbose `using _ = connection;` might be.

**Reply by maxkoshevoi** [reply]
Well, at least this works: `using var _ = connection;`
